addons/website_blog/static/src/js/wysiwyg_adapter.js

```diff
@@ -11,13 +11,6 @@ patch(WysiwygAdapterComponent.prototype, {
         super.init(...arguments);
         this.blogTagsPerBlogPost = {};
     },
-    /**
-     * @override
-     */
-    async startEdition() {
-        await super.startEdition(...arguments);
-        this.options.document.defaultView.$('.js_tweet, .js_comment').off('mouseup').trigger('mousedown');
```

Comment:
@aans-odoo I have removed this code as well, it seems dead code if we remove select to tweet feature.